internal/build/build.go

```diff
@@ -1,2 +1,4 @@
 // Package build contains information about they type of build currently running.
 package build
+
+const DisableThreadChecks = false
```

Comment:
This is always false. There is no build tag?

Reply:
As for a build tag, I'd rather have something like `new-threading-model` or maybe `using-correct-threads` (or similar) over `disable-thread-checks`. The latter sounds a bit negative like you are now using unsafe and might shoot yourself in the foot 😅

Reply:
Hmm, not sure I agree with that, to me the current name is more clear what it's doing, where the `new-threading-model` name suggests it would be enabling something bigger (like if we supported both the old and new threading models together and the app could choose which). And the naming of `using-correct-threads` doesn't really suggest what it does at all IMO.

Reply:
Sorry for the confusion, there is a local TODO in my code maybe i should have left it here ;).
Disabling is a follow up as that is where the naming discussions come in!